src/gui/guimain.cpp

```diff
@@ -234,7 +270,7 @@ void gui_run(void) {
     Screens::Access()->Loop();
 
 #ifdef HAS_RESOURCES
```

Comment:
What is the relation between `#if ENABLED(RESOURCES)` and `#ifdef HAS_RESOURCES`?
As far as I understand, we have an explicit definition of `HAS_RESOURCES` in `CMakeLists.txt`:
```
if(RESOURCES)
  target_compile_definitions(firmware PUBLIC HAS_RESOURCES)
  add_subdirectory(src/resources)
endif()
```
Do we really need the macro `HAS_RESOURCES` from now on?